recipes/configs/dev/qwen3B_full_grpo.yaml

```diff
@@ -116,7 +115,7 @@ num_fsdp_workers: 2
 vllm:
   num_workers: 5
   tp_size: 1
-  batch_size: 1 # TODO: for now, needs to match recipe batch size
+  batch_size: 1
```

Comment:
Keeping these values like this for now, but we need to move away from using 1 as a number... like ANYWHERE :D